pydis_site/apps/content/resources/guides/pydis-guides/how-to-contribute-a-page.md

```diff
@@ -22,17 +22,38 @@ As website changes require staff approval, discussing the page content beforehan
 
 ## Creating the Page
 All pages are located in the `site` repo, at the path `pydis_site/apps/content/resources/`. This is the root folder, which corresponds to the URL `www.pythondiscord.com/pages/`.
+
 For example, the file `pydis_site/apps/content/resources/hello-world.md` will result in a page available at `www.pythondiscord.com/pages/hello-world`.
 
+#### Page Categories
 Nested folders represent page categories on the website. Each folder under the root folder must include a `_info.yml` file with the following:
 
 ```yml
-name: Category name
+title: Category name
 description: Category description
+icon: fas fa-folder # Optional
 ```
 
 All the markdown files in this folder will then be under this category.
 
+#### Having the Category also be a Page
+In order to make categories a page, place a page inside the category folder **with the same name as the category folder**.
+
+```plaintext
+guides
+├── contributing
+│   ├── _info.yml
+│   ├── contributing.md
+│   └── bot.md
+└── _info.yml
+```
```

Comment:
On one hand, it may be more convenient to keep all files related to a category within the same directory. On the other hand, it's less intuitive when considering how normal categories and pages are defined. Have you thought about moving the page out of the directory and instead detecting if the directory contains both a file and subdirectory with the same name (excluding the file extension)?

Reply:
I've thought of that. I personally find putting category pages in the category itself more intuitive.
If `contributing.md` is placed in the `guides` folder, what would be the description shown in the entry for `Contributing` in `..com/guides/`? The current implementation shows the information from `contributing/_info.yml`, while the rendered page has metadata from `contributing.md`.
Would placing `contributing.md` in the `guides` folder then make it confusing as to what information gets displayed where?

Reply:
I see it as less intuitive because it breaks the established convention of a page being a file that is a direct child of the directory. Your point about info coming from different pages follows the same behaviour a normal page and category would follow. Really, the only deviation is that clicking the link in the parent category listing will lead to the page rather than to another category listing. It makes sense to me since I understand that when the parent lists it, it treats it as a normal category in every sense except for what is rendered when the link is clicked.

I don't feel strongly about this, but I think it's worth discussing at least.

Reply:
Also note that with the current approach, it's impossible for it to contain a child page with the same name as the category. Granted, I don't foresee this being something we'd need to do.

Reply:
I see your point. The change won't be hard to make, the main addition is just to not list the category page in the listing if there is also a category with the same name in the directory.